src/gui/footer/footer_items_heaters.cpp

```diff
@@ -98,8 +98,10 @@ FooterItemHeater::HeatState FooterItemHeater::getState(int current, int target,
     }
 
     // preheat mode
-    if ((display > target) && (std::abs(preheat - current) <= heating_difference)) {
-        state = HeatState::preheat;
+    if (display > (target + heating_difference)) {
+        if ((std::abs(temp_diff) <= heating_difference)) {
```

Comment:
Jestli to chápu dobře, "heating_difference" je ve skutečnosti tolerance přesnosti teploty. Tahle podmínka tedy znamená že "dosažení cílové teploty" - abs(temp_diff) <= heating_difference se bude reportovat jako preheating. Určitě to tak má být?

Reply:
musi byt splneno
`display > (target + heating_difference)`
znamena to, ze displej zobrazuje jinou teplotu, nez na kterou se reguluje

Reply:
:ok: